platformio.ini

```diff
@@ -562,6 +562,23 @@ extra_scripts = buildroot/share/PlatformIO/scripts/mks_robin_nano35.py
 lib_deps      = ${common_stm32f1.lib_deps}
   MKS-LittlevGL=https://github.com/makerbase-mks/MKS-LittlevGL/archive/master.zip
 
+#
+# MKS Robin Nano V2 (STM32F103VET6) - MKS UI (LVGL)
+#
+[env:mks_robin_nano35_V2]
+platform      = ststm32
+board         = genericSTM32F103VE
+platform_packages = tool-stm32duino
+build_flags   = !python Marlin/src/HAL/STM32F1/build_flags.py
+  ${common.build_flags} -std=gnu++14 -DHAVE_SW_SERIAL -DSS_TIMER=4
```

Comment:
There seems to be no difference between this environment and the `mks_robin_nano35` environment. The only change is the python script filename, but the script contents are identical.

I suggest deleting the new python script, and removing the new platformio environment.

Reply:
It looks like "nano35" actually is nano V2. Does this need both?
```cpp
#elif MB(MKS_ROBIN_NANO)
  #include "stm32f1/pins_MKS_ROBIN_NANO.h" // STM32F1  env:mks_robin_nano env:mks_robin_nano35
```

Reply:
The pins files are actually somewhat unique. It looks like the V2 uses an I2C EEPROM instead of FLASH, plus a few other minor changes.

Reply:
@thinkyhead: The Nano V2 can probably use both `mks_robin_nano` and `mks_robin_nano35` environments (like the Nano V 1.2) depending on if the user wants to use the emulated 12864/DOGM or LVGL user interface.